empirical-results.tex

```diff
@@ -1 +1,29 @@
-\section{Empirical Results}
\ No newline at end of file
+\section{Empirical Results}
+
+\newcommand{\rref}[1]{}
+
+In order to evaluate our consumption mechanisms, we implement the federated consumption mechanism in Solidity. We provide a concrete implementation of the \textsf{burn-verifier} contract described in \rref{alg.burn-verifier} as \textsf{BurnedBTC}. We implement the \textsf{Crosschain} parent contract from \cite{pow-sidechains}. In order to achieve this we implement our own verification methods similar to \rref{alg.verify-event-federation} in our \textsf{Verifier} library. Specifically, we verify transaction data in \textsf{Verifier} similarly to \rref{alg.verify-tx} by making use of the open-source bitcoin-spv library \cite{bitcoin-spv-library}. We verify block connection using a Merkle Mountain Range where the leaf corresponding to each block has the value of the block's transaction id merkle tree root.
+
+Finally, the \textsf{BurnVerifier} library provides a concrete implementation for our berning protocol. Specifically the library implements the \textsf{BurnVerify} function of the protocol.
+
+In our implementation, the federation is limited to one party for simplicity but it can be easily adapted for more.
```

Comment:
This seems like it could have an impact on gas costs and overall appears like a serious limitation not representative of the federation approach. A reviewer would find this peculiar. Why don't we have a federation of, say, 3?